README.md

```diff
@@ -2,11 +2,48 @@
 
 Qt SDK Creator - A tool for automatically downloading, building and stripping down Qt
 
-## Examples
```

Comment:
I'd rather not have the examples section removed outright.

Reply:
I've restored that